src/hooks/screen/useChatInfoHook.ts

```diff
@@ -51,17 +51,7 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   };
 
   const onContactPressed = (item: ChannelListMemberSchema) => {
-    if (params.from === SIGNED) {
-      return handlePressContact(item);
-    }
-
-    return navigation.push('OtherProfile', {
-      data: {
-        user_id: myUserId?.myProfile?.user_id,
-        other_id: item?.user_id || item?.userId,
-        username: item?.user?.name || item?.user?.username
-      }
-    });
+    return handlePressContact(item);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of `onContactPressed` function seems to have removed the condition that checks if `params.from === SIGNED`. This could potentially lead to undesired behavior as the function now always calls `handlePressContact(item)`, regardless of whether `params.from` is `SIGNED` or not. If this change was intentional and `handlePressContact(item)` is designed to handle all cases, then it's fine. Otherwise, you might want to reintroduce the condition.

```diff
- return handlePressContact(item);
+ if (params.from === SIGNED) {
+   return handlePressContact(item);
+ } else {
+   // Handle other cases here
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->